docs/modules/operation/pages/deep-dive/topology/enlinkd/layer-2/cdp-discovery.adoc

```diff
@@ -1,138 +1,140 @@
 
-= CDP Discovery
+= Cisco Discovery Protocol
 
-The Cisco Discovery Protocol (CDP) is a proprietary link layer protocol from Cisco.
-Network devices use it to advertise their identity, capabilities, and neighbors.
-CDP performs functions similar to several proprietary protocols, such as the Link Layer Discovery Protocol (LLDP), Extreme Discovery Protocol, Foundry Discovery Protocol (FDP), Nortel Discovery Protocol (also known as SONMP), and Microsoft's Link Layer Topology Discovery (LLTD).
-The CDP discovery uses information provided by the http://www.circitor.fr/Mibs/Html/C/CISCO-CDP-MIB.php[CISCO-CDP-MIB] and link:http://www.circitor.fr/Mibs/Html/C/CISCO-VTP-MIB.php[CISCO-VTP-MIB].
-You can find generic information about the CDP process in the CDP Information box on the Node Detail Page of the device.
+The https://learningnetwork.cisco.com/s/article/cisco-discovery-protocol-cdp-x[Cisco Discovery Protocol (CDP)] is a proprietary link layer protocol developed by Cisco.
+Network devices use the protocol to advertise their identities, capabilities, and neighbors.
 
-The CDP Topology Updater provides CDP OnmsTopology consolidating CDP data the CDP Collector collects.
-Only full bidirectional connections between two CDP supported devices become edges.
-Node A and Node B are connected by a CDP edge if and only if there is a CDP MIB port connection in Node A to Node B and vice versa.
+CDP functions similarly to several proprietary protocols, such as the Link Layer Discovery Protocol (LLDP), Extreme Discovery Protocol, Foundry Discovery Protocol (FDP), Nortel Discovery Protocol (also known as SONMP), and Microsoft's Link Layer Topology Discovery (LLTD) protocol.
 
-NOTE: Only nodes with a running CDP process can be part of the link discovery.
-      The data is similar to running a `show cdp neighbor` command on the IOS CLI of the device.
-      Linux and Windows servers don't have a CDP process running by default and will not be part of the link discovery.
+CDP uses information provided by the http://www.circitor.fr/Mibs/Html/C/CISCO-CDP-MIB.php[CISCO-CDP-MIB] and http://www.circitor.fr/Mibs/Html/C/CISCO-VTP-MIB.php[CISCO-VTP-MIB] modules.
+You can find generic information about the CDP process in the CDP Information box on any device's Node Detail page.
```

Comment:
```suggestion
You can find generic information about the CDP process in the CDP information box on any device's node detail page.
```

Reply:
Since this is referring to an actual box/page in the UI, should we keep the capitalization? Doing so would keep this in line with the style guide.